tmp/canary-monitor.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: canary-metrics
+  labels:
+    app: strimzi
+spec:
+  selector:
+    matchLabels:
+      strimzi.io/kind: Canary
```

Comment:
I am not sure we should use the `strimzi.io/kind` as label for selecting the canary service. We use that label in the Strimzi world related to existing "kind" so corresponding custom resources (i.e. Kafka, KafkaConnect and so on). Maybe just using the `app: canary` label? @scholzj what's your opinion on this?

Reply:
Agree, we should use label which we have in `020-Deployment.yaml`

Reply:
ok.